db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,33 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
```

Comment:
@detiber i need to make a change on the ldap fields to allow array input.
but besides that you see any info missing over here? can you think of a field we might need in the future?